python/ray/dashboard/modules/event/tests/test_export_task.py

```diff
@@ -0,0 +1,54 @@
+# isort: skip_file
+# ruff: noqa: E402
+import json
+import os
+import sys
+
+import pytest
+
+# RAY_enable_export_api_write env var must be set before importing
+# `ray` so the correct value is set for RAY_enable_export_api_write
+# even outside a Ray driver.
```

Comment:
I think it doesn't need to be before import, it should be just before creating a ray cluster (i.e. ray.init())

Reply:
This comment was relevant for ray job events where we were checking for state transition events before the driver job was created